contentgrid-spring-audit-logging/src/test/java/com/contentgrid/spring/audit/AuditObservationHandlerTest.java

```diff
@@ -0,0 +1,477 @@
+package com.contentgrid.spring.audit;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.contentgrid.spring.audit.event.AbstractAuditEvent;
+import com.contentgrid.spring.audit.event.EntityContentAuditEvent;
+import com.contentgrid.spring.audit.event.EntityItemAuditEvent;
+import com.contentgrid.spring.audit.event.EntityItemAuditEvent.Operation;
+import com.contentgrid.spring.audit.event.EntityRelationAuditEvent;
+import com.contentgrid.spring.audit.event.EntityRelationItemAuditEvent;
+import com.contentgrid.spring.audit.event.EntitySearchAuditEvent;
+import com.contentgrid.spring.audit.test.handler.AggregatingAuditHandler;
+import com.contentgrid.spring.test.fixture.invoicing.InvoicingApplication;
+import com.contentgrid.spring.test.fixture.invoicing.model.Customer;
+import com.contentgrid.spring.test.fixture.invoicing.model.Invoice;
+import com.contentgrid.spring.test.fixture.invoicing.model.Order;
+import com.contentgrid.spring.test.fixture.invoicing.repository.CustomerRepository;
+import com.contentgrid.spring.test.fixture.invoicing.repository.InvoiceRepository;
+import com.contentgrid.spring.test.fixture.invoicing.repository.OrderRepository;
+import com.contentgrid.spring.test.fixture.invoicing.store.CustomerContentStore;
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.content.commons.property.PropertyPath;
+import org.springframework.context.annotation.Bean;
+import org.springframework.data.rest.webmvc.RestMediaTypes;
+import org.springframework.hateoas.MediaTypes;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+@SpringBootTest(
+        classes = {
+                InvoicingApplication.class,
+                AuditObservationHandlerTest.TestConfig.class
+        },
+        properties = {
+                "server.servlet.encoding.enabled=false" // disables mock-mvc enforcing charset in request
```

Comment:
this is unnecessary in this test ?

Reply:
This is necessary; otherwise the API call with `text/uri-list` will get a charset appended, which is not accepted by spring-data-rest

Reply:
:man_facepalming: